nemo/utils/exp_manager.py

```diff
@@ -599,7 +598,8 @@ def check_resume(
         logging.info(f"Resuming from {last_checkpoints[0]}")
         checkpoint = last_checkpoints[0]
 
-    trainer._checkpoint_connector.resume_from_checkpoint_fit_path = str(checkpoint)
+    # PTL 2.0 supports ckpt_path instead of resume_from_checkpoint as the trainer flag
+    trainer.ckpt_path = str(checkpoint)
```

Comment:
Yep! Awesome. It also supports this as a setter, so I recommend replacing all calls and references to `_checkpoint_connector._ckpt_path` with this.

Reply:
Thanks, Incorporated this @awaelchli